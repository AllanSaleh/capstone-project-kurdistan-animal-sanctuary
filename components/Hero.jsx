import React from "react";
import { useTranslation } from "next-i18next";

function Hero() {
  const { t } = useTranslation("homePage");
  return (
    <div className="bg-white w-80 md:w-96 h-96 mb-10 md:mb-32">
      <img src="/paw.png" alt="paw" className="right-0 absolute z-0" />
      <div className=" flex flex-row absolute z-1 opacity-0 md:opacity-100">
        <img src="/cat.png" alt="cat" />
      </div>
      <div className=" text-center mt-32 z-20 right-0 absolute tracking-widest text-xl md:text-3xl px-0 md:px-20 z-4">
        <p className="font-black uppercase text-primary text-5xl mb-8 ">
          {t("homePage.whyAdopt")}
        </p>
        <p className="font-black uppercase text-primary mb-8 ">
          {t("homePage.adoptNow")}
        </p>
        <button
          type="button"
          className=" py-2  px-20 font-semibold text-primary bg-highlight hover:bg-darken"
        >
          {t("homePage.adoptionPage")}{" "}
        </button>
      </div>
    </div>
  );
}

export default Hero;
