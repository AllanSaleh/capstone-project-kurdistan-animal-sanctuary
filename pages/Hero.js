import React from "react";
import "../styles/Home.module.css";

function Hero() {
  return (
    // eslint-disable-next-line react/jsx-filename-extension
    <div>
      <div>
        <img src="/cat.png" alt="cat" className=""/>
        <img src="/paw.svg" alt="paw" />
      </div>
      <div>
        <p>Make a pet happy</p>
        <p>Adopt Now!</p>
        <button type="button">Go to Adoption page</button>
      </div>
    </div>
  );
}

export default Hero;
