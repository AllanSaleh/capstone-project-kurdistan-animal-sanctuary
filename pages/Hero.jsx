import React from "react";
import "../styles/Home.module.css";

function Hero() {
  return (
    <div>
      <div className=" flex flex-row absolute z-1">
        <img src="/cat.png" alt="cat" />
      </div>
      <div className=" text-center mt-64 z-4 right-0 absolute tracking-widest text-3xl px-20 z-4">
        <p className="font-black uppercase text-green-700 ">Make a pet happy</p>
        <p className="font-black uppercase text-green-700">Adopt Now!</p>
        <button
          type="button"
          className=" py-2  px-20 font-semibold text-green-700 bg-yellow-300 hover:bg-yellow-500"
        >
          Go to Adoption page{" "}
        </button>
      </div>
      <img src="/paw.png" alt="paw" className="right-0 absolute z-1" />
    </div>
  );
}

export default Hero;
