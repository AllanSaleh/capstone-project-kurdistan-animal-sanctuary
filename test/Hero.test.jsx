import React from "react";
import renderer from "react-test-renderer";
import { test, expect } from "@jest/globals";
import Hero from "../components/Hero";

test("proparly renders the Hero component", () => {
  const tree = renderer.create(<Hero />).toJSON();
  expect(tree).toMatchSnapshot();
});
