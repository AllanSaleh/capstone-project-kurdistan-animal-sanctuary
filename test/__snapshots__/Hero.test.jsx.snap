// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proparly renders the Hero component 1`] = `
<div
  className="bg-white w-80 md:w-96 h-96 mb-10 md:mb-32"
>
  <img
    alt="paw"
    className="right-0 absolute z-0"
    src="/paw.png"
  />
  <div
    className=" flex flex-row absolute z-1 opacity-0 md:opacity-100"
  >
    <img
      alt="cat"
      src="/cat.png"
    />
  </div>
  <div
    className=" text-center mt-32 z-20 right-0 absolute tracking-widest text-xl md:text-3xl px-0 md:px-20 z-4"
  >
    <p
      className="font-black uppercase text-primary text-5xl mb-8 "
    >
      Make a pet happy
    </p>
    <p
      className="font-black uppercase text-primary mb-8 "
    >
      Adopt Now!
    </p>
    <button
      className=" py-2  px-20 font-semibold text-primary bg-highlight hover:bg-darken"
      type="button"
    >
      Go to Adoption page
       
    </button>
  </div>
</div>
`;
